frontend/packages/kubevirt-plugin/src/k8s/wrapper/vm/vmi-wrapper.ts

```diff
@@ -30,7 +30,7 @@ export class VMIWrapper extends K8sResourceWrapper<VMIKind, VMIWrapper> implemen
   getOperatingSystem = () => findKeySuffixValue(this.getLabels(), TEMPLATE_OS_LABEL);
   getWorkloadProfile = () => findKeySuffixValue(this.getLabels(), TEMPLATE_WORKLOAD_LABEL);
   getFlavor = () => findKeySuffixValue(this.getLabels(), TEMPLATE_FLAVOR_LABEL);
-
+  getEvictionStrategy = () => this.data.spec?.template?.spec?.evictionStrategy;
```

Comment:
IMHO it will be nice to do, see comment https://github.com/openshift/console/pull/7276/files#r528288742:
```
getVirtualMachineInstanceSpec = (): VMISpec => this.data.spec;
getEvictionStrategy = (): string => getVirtualMachineInstanceSpec()?..evictionStrategy;
```